coderd/database/dbauthz/dbauthz.go

```diff
@@ -170,6 +170,9 @@ var (
 					rbac.ResourceWorkspaceBuild.Type: {rbac.ActionRead, rbac.ActionUpdate, rbac.ActionDelete},
 					rbac.ResourceUserData.Type:       {rbac.ActionRead, rbac.ActionUpdate},
 					rbac.ResourceAPIKey.Type:         {rbac.WildcardSymbol},
+					// Might want to reduce the org read scope to a specific org
+					// in the future.
```

Comment:
let's make this comment more concrete:  once we have org-scoped provisionerd credentials, we can reduce read scope to a specific org